src/renderer/components/app.tsx

```diff
@@ -86,6 +88,15 @@ export class App extends React.Component {
 
     await requestMain(clusterSetFrameIdHandler, clusterId);
     await getHostedCluster().whenReady; // cluster.activate() is done at this point
+
+    const activeEntityDisposer = reaction(() => catalogEntityRegistry.getById(clusterId), (entity) => {
```

Comment:
Maybe more reasonable to move this disposer code into `catalogEntityRegistry.init()` as it relates to registry itself.